source/agora/common/Config.d

```diff
@@ -149,10 +149,13 @@ public struct InterfaceConfig
     /// Bind port
     public ushort port;
 
+    /// Whether to use tls for this interface
+    public bool tls = false;
+
     /// Default values when none is given in the config file
     private static immutable InterfaceConfig[/* Type.max */ 1] Default = [
         // Publicly enabled by default
-        { type: Type.http, address: "0.0.0.0", port: 0xB0A, },
+        { type: Type.http, address: "0.0.0.0", port: 0xB0A, tls: false },
```

Comment:
Wouldn't it make more sense to add a new `Type`, e.g. `https` ?

Reply:
Yes, I'll update.